flotiq/flotiq-codegen-ts
# #24775 changed bash scripts to js scripts

**Refactored:**
- changed all .`sh` scripts to `.js` scripts

**Testing:**
 - run `yarn start` should generate typescipt package
 - run `yarn start --compiled-js` should generate compiled typescipt to js package
